### DISCUSSION TITLE
Control workspace that an npm module is installed in?

### DISCUSSION THREAD

**Discussion by jamezening** [vercel/turborepo#5188] [Help] [answered]
Hi,

I have a standard mono repo setup like

(ROOT)
node_modules
apps
|-- {myExpoApp}
     node_modules
|-- {anotherApp}
packages
|-- {package1}
|-- {package2}
...

myExpoApp is an Expo app.
I installed the relevant packages in the apps/myExpoApp directory using npx expo / npm.

PROBLEM: one of the packages (`expo`) installed to `node_modules` in (ROOT) is importing as follows:
`import App from '../../App';`

This would work if the npm modules were to have been installed in the apps/myExpoApp/node_modules folder, but it is importing from (ROOT).

Is there a Turborepo or other configuration I can use to workaround this?

My expo project is [setup for monorepos](https://docs.expo.dev/guides/monorepos/) and I have tried the
[Install a package in a workspace](https://turbo.build/repo/docs/handbook/package-installation#addingremovingupgrading-packages) link the Turborepo documentation, but even when tyring to install the package (`expo`) to myExpoApp workspace, it gets installed to (ROOT).

Thank you for any comments and help.

**Comment by anthonyshew** [comment, marked as answer]
Depending on your package manager, package manager configuration, and dependencies, your node_modules will be organized differently from project to project. Another thing to note is that it _may or may not_ be a problem that Expo is being installed to and resolved to the root. If it works, it works. 🤷 

Turborepo sits on top of your package manager's workspaces so doesn't have any responsibility here!
